lib/util.js

```diff
@@ -149,15 +149,17 @@ const util = {
     const braveAppVectorIconsDir = path.join(config.projects['brave-core'].dir, 'vector_icons', 'chrome', 'app')
     const chromeAndroidDir = path.join(config.srcDir, 'chrome', 'android')
     const braveAndroidDir = path.join(config.projects['brave-core'].dir, 'android')
-    const braveChromiumSrcDir = path.join(config.projects['brave-core'].dir, 'chromium_src')
 
     let fileMap = new Set();
     // The following 3 entries we map to the same name, not the chromium equivalent name for copying back
     autoGeneratedBraveToChromiumMapping[path.join(braveAppDir, 'brave_strings.grd')] = path.join(chromeAppDir, 'brave_strings.grd')
     autoGeneratedBraveToChromiumMapping[path.join(braveAppDir, 'settings_brave_strings.grdp')] = path.join(chromeAppDir, 'settings_brave_strings.grdp')
-    autoGeneratedBraveToChromiumMapping[path.join(braveAppDir, 'components_brave_strings.grd')] = path.join(config.srcDir, 'components', 'components_brave_strings.grd')
+    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'components_brave_strings.grd')] = path.join(config.srcDir, 'components', 'components_brave_strings.grd')
+    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'components_strings.grd')] = path.join(config.srcDir, 'components', 'components_strings.grdp')
+    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'management_strings.grdp')] = path.join(config.srcDir, 'components', 'management_strings.grdp')
+    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'password_manager_strings.grdp')] = path.join(config.srcDir, 'components', 'password_manager_strings.grdp')
+    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'bookmark_bar_strings.grdp')] = path.join(config.srcDir, 'components', 'bookmark_bar_strings.grdp')
```

Comment:
Nit: `chromeComponentsDir` is defined above.

Reply:
Fixed.